samps/src/TLE.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+//
+// File:    TLE.java
+// Purpose: enters an infinite with no output, forcing a Time Limit Exceeded result.
+//
+
+public class TLE {
+    
+    public static void main(String[] args) {
+        int i = -1;
+        while (i < 1) {
+            //do nothing
+        }
```

Comment:
Hmmm... I have a feeling that I might have been the original author of this program, but:  is it possible that the Java compiler will optimize-out the above while-loop?  (I've seen C compilers that would do that...)

Reply:
Nope.  I tried it on the latest image.  The compiler doesn't even issue a warning about it, and, it runs forever...